src/main/java/uk/gov/hmcts/darts/courthouse/controller/CourthousesController.java

```diff
@@ -36,6 +39,8 @@ public class CourthousesController implements CourthousesApi {
 
     private final CourthouseToCourthouseEntityMapper mapper;
```

Comment:
Since we now have two mappers, it's worth renaming this variable to `courthouseMapper` for clarity.

Reply:
Agreed - I had already done this for the unit test. Updated controller as suggested.